extensions/guacamole-auth-cas/src/main/resources/translations/ca.json

```diff
@@ -0,0 +1,12 @@
+{
+
+    "DATA_SOURCE_CAS" : {
+        "NAME" : "CAS SSO Backend"
```

Comment:
If this is the proper translation, it should be kept, but if you are not translating this, it should be removed.

Reply:
I improve the translation